index.js

```diff
@@ -28,7 +31,8 @@ const BlueAppComponent = () => {
 
   return (
     <BlueStorageProvider>
-      <App />
+      {wasmInitializationFinished && <App />}
+      <WasmWebView setWasmInitializationFinished={setWasmInitializationFinished} />
```

Comment:
Please check on device that webview is not crashing because of network and non-successful wasm loading is not prevent app from loading. At least btc part have to work

Reply:
I have checked it on real device with release build.

- webview is not crashing because of network
- non-successful wasm loading is not prevent app from loading, since when webview load fails, the onLoadEnd function is also called